tools/dataset_converters/acdc.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+import os.path as osp
+import shutil
+
+from mmengine.utils import mkdir_or_exist
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert ACDC dataset to mmsegmentation format')
+    parser.add_argument('raw_data', help='the path of raw data')
+    parser.add_argument(
+        '-o', '--out_dir', help='output path', default='./data/acdc')
+    parser.add_argument(
+        '--split',
+        choices=['fog', 'night', 'rain', 'snow', 'all'],
+        default='night')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+
+    try:
+        shutil.rmtree(args.out_dir)
+        print('Directory removed successfully')
+    except OSError as e:
+        print(f'Error: {e.strerror}')
```

Comment:
This is a dangerous operation and it is best not to delete any of the user's files.

Reply:
Updated!